### ISSUE TITLE
Consider the introduction of a new header for files

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#1401]
The current header lines were defined 20+ years ago.
There have been minor augmentations in the past 3 years to permit use of +LDS and more recently the #,DEF for flexible_csv.
The vast majority of the old fields are now redundant. This gives rise to significant confusion.
Perhaps it is time to define a new simple header.

**Comment by Sherlock21** [comment]
Yes. I agree with this suggestion.

**Comment by suffolkroots** [comment]
I agree with this suggestion. Would you envisage the new simplified header seamlessly working with existing versions of WinREG/WinfreeREG? I ask as support for these apps is less than it was.

**Comment by SteveBiggs** [comment]
@suffolkroots, Mike Fry is back in business and will be picking up where he left off with getting WinFreeREG finished. This will include the flexible_csv support and simplified headers if that is implemented. WinREG will be retired at that point so will not be modified to support these new features.

**Comment by suffolkroots** [comment]
Thanks for the udate Steve. 
So pleased that Mike Fry is back.
